### ISSUE TITLE
new peak shape for MJD

### ISSUE THREAD

**Issue by sofia-calgaro** [sofia-calgaro/ZeroNuFit.jl#51]
Implement gaussian+low-E side tail for MJD partitions.
Idea: add a key in the partition.json (under `fit_group`?) for specifying the peak shape to use for partitions listed in that file, eg `"signal_shape": "gaussian"` or `"signal_shape": "gaussian_plus_lowEtail"`.
In this way, we can handle differently different experiments.

**Comment by tdixon97** [comment]
Yes of course that works, although it becomes a bit 'hardcoded' but idk a better way to do this..

**Comment by sofia-calgaro** [comment]
the only not-hardcoded way would be via the config.json, but I think there you lose some freedom... There you could specify something like:

```
        "events": [
            "legend-0vbb-config/events_gerda.json",
            "legend-0vbb-config/events_l200.json",
            "legend-0vbb-config/events_mjd.json",
        ],
        "partitions": [
            "legend-0vbb-config/partitions_gerda.json",
            "legend-0vbb-config/partitions_l200.json",
            "legend-0vbb-config/partitions_mjd.json"
        ],
        "signal_shape": [
            "gaussian",
            "gaussian",
            "gaussian_plus_lowEtail"
        ],
```

Anyway, we can set the `gaussian` as the default shape and let the user specify only when the other shape has to be used

**Comment by tdixon97** [comment]
yeah i meant that ideally one could input directly a formula, but its very hard to implement, i think it belongs in the partitions file not the config file?

**Comment by tdixon97** [comment]
so i agree with your original suggestion, its just a pity that you have to rely on names for shapes, but idk a way around it (similar for the background shapes).

**Comment by sofia-calgaro** [comment]
the new peak shape works after https://github.com/sofia-calgaro/ZeroNuFit.jl/commit/3d46a628329bc30a52990b67930e760e57425866 and was tested

- succesfully for MJD only
- succesfully for L200 only (where we don't add any gamma/frac/tau parameter and don't specify `signal_shape`
- succesfully for the combination MJD+L200

However, in a combined fit, the fit does not work if we change the order, eg if we fit L200+MJD (this is related to the number of partitions for which we want to include the prior gamma ... this has to be handled better...)

**Comment by sofia-calgaro** [comment]
Missing features:
- add data+model plot with modified peak shape for given partitions
